packages/client/src/components/TrecipeCard/TrecipeCard.tsx

```diff
@@ -1,20 +1,24 @@
 import React from 'react';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import './trecipeCard.scss';
-import { CardMenu } from '../../../components/CardMenu/CardMenu';
-import { MenuItem } from '../../../components/Menu/Menu';
-import { ProgressBar } from '../../../components/ProgressBar/ProgressBar';
+import { CardMenu } from '../CardMenu/CardMenu';
+import { MenuItem } from '../Menu/Menu';
+import { ProgressBar } from '../ProgressBar/ProgressBar';
 import { bindActionCreators, Dispatch } from 'redux';
-import { deleteTrecipeRequest, duplicateTrecipeRequest } from '../../../redux/TrecipeList/action';
+import { deleteTrecipeRequest, duplicateTrecipeRequest } from '../../redux/TrecipeList/action';
 import { connect } from 'react-redux';
-import { showModal } from '../../../redux/Modal/action';
-import TrecipePopup, { TrecipePopupType } from '../../../components/TrecipePopup/TrecipePopup';
-import Trecipe from '../../../../../shared/models/trecipe';
-import { baseURL } from '../../../api';
+import { showModal } from '../../redux/Modal/action';
+import TrecipePopup, { TrecipePopupType } from '../TrecipePopup/TrecipePopup';
+import Trecipe from '../../../../shared/models/trecipe';
+import { baseURL } from '../../api';
 
 type TCProps = Trecipe & ReturnType<typeof mapDispatchToProps>;
 
```

Comment:
@tianyan-zhu If you defined a TCOwnProps consisting of a Trecipe type field and `isReadOnly` field, I don't think it will cause drastically big changes?